.github/workflows/danger.yml

```diff
@@ -32,10 +34,14 @@ jobs:
         # frozen-lockfile will make the build fail if the lockfile is not there
         run: yarn install --frozen-lockfile
 
-      - name: Check
+      - name: Validate Labels
+        run: |
+          yarn run danger ci \
+            --dangerfile Automattic/peril-settings/org/pr/label.ts \
+            --id pr_labels
```

Comment:
A previous version of this had a `--no-publish-check` parameter, which [I thought was cool because it makes the checks component in the PR tidier](https://github.com/mokagio/WordPress-iOS/pull/25#discussion_r381089436).

What I didn't notice at the time was that if we don't publish the check, Danger is not able to fail the build.

I'm not sure whether that's a bug or a feature, I'll have to investigate. I do think that having a bit more checks in the list is totally worth the safety of Danger failing the build.